apps/workplace/src/app/book/new-desk-flow/new-desk-flow-confirm.component.ts

```diff
@@ -0,0 +1,163 @@
+import { Component, Input, Optional } from '@angular/core';
+import { MatBottomSheetRef } from '@angular/material/bottom-sheet';
+import { BookingFormService } from '@placeos/bookings';
+import { BaseClass } from '@placeos/common';
+import { Desk, OrganisationService } from '@placeos/organisation';
+
+@Component({
+    selector: 'new-desk-flow-confirm',
+    styles: [
+        `
+            section > app-icon {
+                font-size: 1.5rem;
+                margin-top: 0.3rem;
+            }
+
+            h2 {
```

Comment:
This could be inline as there is only one h2 tag.

Reply:
Changed to inline